### DISCUSSION TITLE
Blur background

### DISCUSSION THREAD

**Discussion by HanaO00** [jellyfin/jellyfin-androidtv#4249] [General]
Hello, is it possible to disable blur on backdrops?
The version on the play store does not have blur but the latest versions on github have it and I can't find it in the settings

**Comment by nielsvanvelzen** [comment]
You can enable/disable backdrops. There are no additional preferences to how they display, the blurring of the images is not available on all devices (depends on some device capabilities). There are no plans to add settings for it.

**Reply by HanaO00** [reply]
Thanks for your reply !
